scrollytelling/src/primitive.tsx

```diff
@@ -471,18 +479,18 @@ function getValidAt(at: number) {
 interface ParallaxProps {
   tween: Omit<TweenBaseDef, "to" | "from" | "fromTo"> & {
     target?: TweenTarget; // Optional: The target element or elements to apply the animation to.
-    movementX?: UnitValue; // Optional: The amount of movement on the X-axis.
-    movementY?: UnitValue; // Optional: The amount of movement on the Y-axis.
-  };
+  } & RequireAtLeastOne<{
+    movementX?: UnitValue; // The amount of movement on the X-axis.
+    movementY?: UnitValue; // The amount of movement on the Y-axis.
+  }>;
   children?: React.ReactNode; // Optional: Content to be rendered inside the Parallax component.
 }
 
 /**
- * Parallax component applies a parallax effect to its children using GSAP animations.
- *
- * @param {ParallaxProps} props - Parallax component props
```

Comment:
Removed this line since it doesn't add much value and adding all the type explanation here would make it too verbose. Based on other component libraries I think is best to keep these comments short and (if necessary) add more reference links.